### PR TITLE
Moved quantity calculation to its own method `calculateTotalQuantity()`

### PR DESCRIPTION
**Pull request by shineability**
- Moved quantity calculation to its own method `calculateTotalQuantity()`
- CS fixes

@sprankhub I removed the underscores from the protected functions (PSR-2), hope this is ok for you? Personally, I hate them, but I know it's still common practice in M1... Feel free to add them again :)

### CODE REVIEW

**Comment by shineability** [comment]
@sprankhub Failed checks don't seem to be related to the actual tests. Can you have a look?

**Comment by sprankhub** [comment]
Sorry that it takes so long, but since this is a critical change and involves background processes (new release, update on Marketplace etc.), we need some more time. I definitely have it on my list and will check this PR as soon as possible.

File: app/code/community/CustomGento/ConfigurableTierPrices/Helper/Data.php [hunk 1]
```diff
@@ -2,11 +2,8 @@
 
 class CustomGento_ConfigurableTierPrices_Helper_Data extends Mage_Core_Helper_Abstract
 {
-
     const XML_PATH_IS_ENABLED = 'sales/customgento_configurabletierprices/is_enabled';
     const XML_PATH_DISABLED_FOR_CATEGORY = 'sales/customgento_configurabletierprices/disabled_for_category';
-
-    // attribute code length must be less than 30 symbols!
```
Comment (sprankhub) [inline app/code/community/CustomGento/ConfigurableTierPrices/Helper/Data.php]: Please keep this comment (as it is non-intuitive to use an abbreviated version of the extension name here).

File: app/code/community/CustomGento/ConfigurableTierPrices/Helper/Data.php [hunk 2]
```diff
@@ -41,15 +39,15 @@ public function isProductInDisabledCategory(Mage_Catalog_Model_Product $product)
 
     public function isExtensionDisabledForProduct(Mage_Catalog_Model_Product $product)
     {
-        // get the product attribute directly, because it may not be loaded
+        // Get the product attribute directly, because it may not be loaded
```
Comment (sprankhub) [inline app/code/community/CustomGento/ConfigurableTierPrices/Helper/Data.php]: Please do not change such subtle things in a PR. It makes it harder to merge (and I prefer starting with lower case letters btw).

File: app/code/community/CustomGento/ConfigurableTierPrices/Model/Observer.php [hunk 3]
```diff
@@ -2,33 +2,48 @@
 
 class CustomGento_ConfigurableTierPrices_Model_Observer
 {
+    /**
+     * @var CustomGento_ConfigurableTierPrices_Helper_Data
+     */
+    protected $helper;
```
Comment (sprankhub) [inline app/code/community/CustomGento/ConfigurableTierPrices/Model/Observer.php]: Please rename to `$_helper`.

File: app/code/community/CustomGento/ConfigurableTierPrices/Model/Observer.php [hunk 4]
```diff
@@ -38,56 +53,55 @@ public function catalogProductGetFinalPrice(Varien_Event_Observer $observer)
...
+     * Calculate product final price via configurable product's tier pricing structure.
+     *
+     * Uses quantity of parent item to determine price.
      *
      * @param   Mage_Catalog_Model_Product $product
      *
      * @return  float
      */
-    protected function _calcConfigProductTierPricing($product)
+    protected function calculateTierPrice($product)
```
Comment (sprankhub) [inline app/code/community/CustomGento/ConfigurableTierPrices/Model/Observer.php]: Please rename to `_calculateTierPrice`.

File: app/code/community/CustomGento/ConfigurableTierPrices/Model/Observer.php [hunk 5]
```diff
@@ -38,56 +53,55 @@ public function catalogProductGetFinalPrice(Varien_Event_Observer $observer)
...
-                $totalQty  = array_sum($idQuantities[$product->getId()]);
-                $tierPrice = $product->getPriceModel()->getBasePrice($product, $totalQty);
+    /**
+     * Calculate total quantity of product in cart.
+     *
+     * @param  Mage_Catalog_Model_Product $product
+     *
+     * @return int
+     */
+    protected function calculateTotalQuantity($product)
```
Comment (sprankhub) [inline app/code/community/CustomGento/ConfigurableTierPrices/Model/Observer.php]: Please rename to `_calculateTotalQuantity`.

File: app/code/community/CustomGento/ConfigurableTierPrices/Model/Observer.php [hunk 6]
```diff
@@ -38,56 +53,55 @@ public function catalogProductGetFinalPrice(Varien_Event_Observer $observer)
...
+        });
     }
 
     /**
      * Retrieves all visible quote items from the session.
      *
      * @return array with instances of Mage_Sales_Model_Quote_Item
      */
-    protected function _getAllVisibleItems()
+    protected function getAllVisibleItems()
```
Comment (sprankhub) [inline app/code/community/CustomGento/ConfigurableTierPrices/Model/Observer.php]: Please revert to `_getAllVisibleItems`.

**Review by sprankhub — CHANGES_REQUESTED** [review]
Sorry for the late feedback. I had a quick look at it now and have to request a few changes. We will process the PR further once they are corrected. Thanks!

**Comment by sprankhub** [comment]
Thanks for the quick update! My colleague @norgeindian will further test and process this PR.

**Comment by shineability** [comment]
@norgeindian Tnx!

**Pull request merged**